driver_go18_test.go

```diff
@@ -98,6 +106,118 @@ func getQueryProcess(processes []dbProcess, query string) *dbProcess {
 	return nil
 }
 
+func pollQueryProcess(db *sql.DB, dbName, query string, totalTimeout, pollTimeout time.Duration) (*dbProcess, error) {
+	end := time.Now().Add(totalTimeout)
+	var (
+		err error
+		processList []dbProcess
+		longProcess *dbProcess
+	)
+	for time.Now().Before(end) {
+		processList, err = getProcesslist(dbName, db)
+		longProcess = getQueryProcess(processList, query)
+
+		if err != nil || longProcess == nil {
+			time.Sleep(pollTimeout)
+		} else {
+			break
+		}
+	}
+	if longProcess == nil {
+		return nil, fmt.Errorf("process for query \"%s\" not found", query)
+	}
+	return longProcess, err
+}
+
+func pollKillProcess(db *sql.DB, dbName string, processID int, totalTimeout, pollTimeout time.Duration) error {
```

Comment:
does not need, since it is used only in one time.

Reply:
Removed (initially created this function to prevent killQuery from being too big)